Tools/gdb/libpython.py

```diff
@@ -1414,13 +1410,13 @@ def proxyval(self, visited):
         elif repr_kind == 4:
             field_str = field_str.cast(_type_unsigned_int_ptr())
 
-        # Gather a list of ints from the Py_UNICODE array; these are either
+        # Gather a list of ints from the character array; these are either
         # UCS-1, UCS-2 or UCS-4 code points:
-        Py_UNICODEs = [int(field_str[i]) for i in safe_range(field_length)]
+        characters = [int(field_str[i]) for i in safe_range(field_length)]
 
         # Convert the int code points to unicode characters, and generate a
         # local unicode instance.
-        result = u''.join(map(chr, Py_UNICODEs))
+        result = u''.join(map(chr, characters))
```

Comment:
```suggestion
        result = ''.join(map(chr, characters))
```

Reply:
Removing supporting for Python 2 require way more changes. I prefer to restrict changes to just Py_UNICODE here.

Reply:
FYI, I removed Python 2 support from libpython.py already.
https://github.com/python/cpython/pull/31717/files